packages/wdio-mocha-framework/src/index.ts

```diff
@@ -43,6 +43,7 @@ class MochaAdapter {
     private _suiteCnt: Map<string, number> = new Map()
     private _hookCnt: Map<string, number> = new Map()
     private _testCnt: Map<string, number> = new Map()
+    private _suiteStartDate!: Date
```

Comment:
Let's initiate it with a value so we don't need to worry about it not being defined.
```suggestion
    private _suiteStartDate: number = Date.now()
```